src/FSharp.Core/printf.fs

```diff
@@ -852,11 +872,19 @@ module internal PrintfImpl =
     
     module FloatAndDecimal = 
 
+        let fixupDecimalSign (n: decimal) (nStr: string) =
```

Comment:
`decimal` not including the negative sign for 0 is an intentional choice for this type

Reply:
@vzarytovskii we should probably just follow the same behavior as .NET here then, right? i.e. make `-0` decimals format the same as `+0` decimals

Reply:
I would ping @t-gro for that, let the team decide what's desired and whether suggestion is needed.

Reply:
It is a change we could take - but it should be documented on the .NET breaking changes and happen on a major version update.